### ISSUE TITLE
User can only see a subset of entries

### ISSUE THREAD

**Issue by brindakv** [informatics-isi-edu/protein-database#121]
`pdb-submitter` is only able to see a subset of entries on `dev` (7 out of 23). Unable to test this on production or staging because the test user does not have depositions on production or staging. 

This happens on Google chrome Version `108.0.5359.124` on MacOS Ventura 13.1.  Doesn't happen on other computers. 

Update: The 7 entries are not created by the user. The user is not able to see any of his/her entries and is able to see entries created by a different user. 

To summarize: 

As user on dev: 7 entries created by the the curator (1-NY68, 1-NVT6, 1-NPHY, 1-NKDJ, 1-N4XR, 1-N4XP, and 1-N4XM) are seen by the user. 0 of the 23 entries created by the user are seen by the user. 

As user of staging: No entries are seen although there is 1 entry created by the user. 

As user on production: No entries have been created by the test user on production, and not entries are seen.

**Comment by hongsudt** [comment]
Did you try this on ff or on a different machine? Can you share your screen
shot?

I used your test account to check on my browser and saw the result
consistent with the policy, all 23.

**Comment by brindakv** [comment]
@hongsudt did you test both `dev` and `staging`?

**Comment by hongsudt** [comment]
I only tested this on dev, which should have the same acl policies as prod.

 I don't think we set up staging eg. We didn't replicate the prod content
to staging (the step that we should all enforce from now on) and I am not
sure serban applied policies to staging.

Maybe Monica can help test this?

**Comment by brindakv** [comment]
Testing from firefox on a different computer, user is able to all `23` entries on `dev` and a single entry on `staging`. The ACLs for accession code look good for these entries (user is able to see accession codes for those entries with HOLD and REL status #118). 

The issue seems to be only on chrome.
